frontend/src/hooks/use-query.ts

```diff
@@ -0,0 +1,7 @@
+import { useLocation } from 'react-router';
```

Comment:
Can we just make this useQuery? We don't have - in any of our other file names as far as I'm aware.